static/.htaccess

```diff
@@ -8,10 +8,12 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Remove OSS section
+RewriteRule ^docs/components/zeebe/open-source(.*)$ /docs/components/zeebe/zeebe-overview/ [R=301,L]
```

Comment:
I keep thinking "wow, I'm surprised we don't need a rule like this for the other versions," and then remembering that only the current version is in the sitemap. 🙌

Reply:
YES!!! What a win for cognitive load!